WTWSMS/common/cb_types/WtWSMS_migration_unification_cb_types.txt

```diff
@@ -39,12 +39,23 @@ migration = {
 				ROOT = {
 					OR = {
 						religion = tengri_pagan
+						religion = tengri_pagan_reformed
 						religion = slavic_pagan
+						religion = slavic_pagan_reformed
 						religion = germanic_pagan
+						religion = germanic_pagan_reformed
 						culture = alan
+						is_nomadic = yes
 					}
 					NOT = { has_character_modifier = launched_migration_war }
-					NOT = { higher_tier_than = KING } # No Emperors
+
+					NOT = {
+						AND = {
+							higher_title_than = KING
+							is_nomadic = no
+						}
+					}
+	
 					NOT = { culture = FROM }
 					independent = yes # No vassals
 					war = no # Ensures that the events activate correctly  
```

Comment:
@loup99 

SHOOT! Loup, I'm so sorry. I didn't mean to disable it for everyone except nomads.

What I was trying to code was this:

1. Nomads can migrate, but no other type of Emperor can.
2. Reformed incarnations of the previously available nomads can still migrate. Why should a Slavic or Germanic/Norse Pagan stop doing what they do best?

For the sake of historical accuracy, I also think that Magyars, Iazygens, and Iranian Steppe Pagans should be allowed to migrate freely (as they did in RL), but I decided to save that for another time.